tests/foreman/cli/test_contenthost.py

```diff
@@ -567,115 +559,38 @@ def setUp(self):
 
```

Comment:
Actually the behaviour will depend on the test runner, on py.test:

```
py.test test.py
=================================== test session starts ===================================
platform darwin -- Python 2.7.6 -- py-1.4.26 -- pytest-2.6.4
collected 2 items

test.py EE

========================================= ERRORS ==========================================
_______________________________ ERROR at setup of A.test_1 ________________________________

cls = <class 'test.A'>

    @classmethod
    def setUpClass(cls):
>       raise Exception('Setup class failed')
E       Exception: Setup class failed

test.py:6: Exception
_______________________________ ERROR at setup of A.test_2 ________________________________

cls = <class 'test.A'>

    @classmethod
    def setUpClass(cls):
>       raise Exception('Setup class failed')
E       Exception: Setup class failed

test.py:6: Exception
================================= 2 error in 0.03 seconds =================================
```

On nosetests:

```
nosetests test.py
E
======================================================================
ERROR: test suite for <class 'test.A'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/elyezer/.virtualenvs/robottelo/lib/python2.7/site-packages/nose/suite.py", line 209, in run
    self.setUp()
  File "/Users/elyezer/.virtualenvs/robottelo/lib/python2.7/site-packages/nose/suite.py", line 292, in setUp
    self.setupContext(ancestor)
  File "/Users/elyezer/.virtualenvs/robottelo/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/Users/elyezer/.virtualenvs/robottelo/lib/python2.7/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/Users/elyezer/code/robottelo/test.py", line 6, in setUpClass
    raise Exception('Setup class failed')
Exception: Setup class failed

----------------------------------------------------------------------
Ran 0 tests in 0.016s

FAILED (errors=1)
```

Sample test module used:

```python
import unittest

class A(unittest.TestCase):
    @classmethod
    def setUpClass(cls):
        raise Exception('Setup class failed')

    def test_1(self):
        self.fail('test_1')

    def test_2(self):
        self.fail('test_2')
```